src/ResourceManager/Network/Commands.Network/VirtualNetwork/NewAzureVirtualNetworkCommand.cs

```diff
@@ -88,11 +89,14 @@ public class NewAzureVirtualNetworkCommand : VirtualNetworkBaseCmdlet
             HelpMessage = "Reference to the DDoS protection plan resource associated with the virtual network.")]
         public string DdosProtectionPlanId { get; set; }
 
+#if !NETSTANDARD
+        [CmdletParameterBreakingChange("EnableVmProtection", ChangeDescription = "The -EnableVmProtection parameter is deprecated and ignored")]
```

Comment:
This should be more informative - customers will use this deprecation to decide what to do with their scripts.  or example: ```The EnableVMProtection setting is no longer supported on the service.  Setting this parameter has no impact on the Vitrual Machine. This parameter will be removed in a future release. Please remove it from your scripts.```